[repo] docs: added Data Sources to getting started docs (#1184)
microsoft/teams-ai: getting-started/CONCEPTS/ACTION-PLANNER.md

The Action Planner has a built-in prompt management system that supports creating prompt templates as folders in the file system. A prompt template is the prompt text along with all the configurations for completion with the LLM model. Dynamic prompts also support template variables and functions.

#### [Data Sources](./DATA-SOURCES.md)

> To augment prompts even further and facilitate better responses, data sources can be used.

Reviewer: ```suggestion
Use data sources to augment prompts even further and facilitate better responses.
```
